packages/client/src/App.sass

```diff
@@ -100,8 +100,8 @@ $modal-background-background-color: rgba(1, 1, 1, 0.5)
 $modal-content-width: 430px
 .modal-content
   max-height: calc(100vh - 210px)
-  overflow-y: auto !important
-  overflow-x: hidden !important
+  overflow-y: visibile !important
+  overflow-x: visible !important
```

Comment:
It's impossible to reach the button if there is no possibility to scroll the modal

Reply:
Sending NFTs on my laptop screen is too short, so button cannot be reached unless I zoom out
There are success screens (after the token/NFT is executed), that are a bit longer, and on mobile view we have to also make sure it's scrollable

Reply:
Got it, that is for sure a problem, I searched for solutions but looks like there's no other css way to show the dropdown while make the modal scrollable. I'll work on a function to set the overflow property dynamically.